<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="seat_A01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F06">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F07">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F04">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F03">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F01">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_E09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>E09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_F08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>F08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_A08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_C08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_D09">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D09</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seat_B08">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_goback">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이전페이지</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>다음페이지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>651</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_moviename">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_movietime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_theater">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_adtnum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>640</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_teennum">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
